srcpkgs/perl-boolean/template

```diff
@@ -0,0 +1,15 @@
+# Template file for 'perl-boolean'
+pkgname=perl-boolean
+version=0.46
+revision=1
+wrksrc="${pkgname/perl-/}-${version}"
```

Comment:
Why not just `boolean` ?

Reply:
Are you asking about pkgname or wrksrc ?

Reply:
Bumped and static'd `boolean` in!